src/js/jsx/shared/Select.jsx

```diff
@@ -47,12 +47,13 @@ define(function (require, exports, module) {
             value: React.PropTypes.shape({
                 id: React.PropTypes.string.isRequired,
                 title: React.PropTypes.string.isRequired,
-                style: React.PropTypes.object
+                info: React.PropTypes.string,
+                style: React.PropTypes.object,
+                svgType: React.PropTypes.string
```

Comment:
Re: my comment above about a utility function for icons, let's keep thinking about whether this is the right way to expose that functionality. Other options: 1) parametrize values with an optional `icon` ReactElement; 2) don't parametrize them at all, but just include this as part of the `style`.